website/vitest.config.js

```diff
@@ -19,7 +19,7 @@
 const config = {
   test: {
     environment: 'node',
-    include: ['*.{test,spec}.?(c|m)[jt]s?(x)'],
+    include: ['**/*.{test,spec}.?(c|m)[jt]s?(x)'],
```

Comment:
you could add this change as a separate PR (merged quicly)

Reply:
https://github.com/podman-desktop/podman-desktop/pull/13230/files